frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua

```diff
@@ -1,514 +1,744 @@
 return {
+    zro = {"", width = 0}, -- zero-wdith key "placeholder" which does not affect other key centering. Can be used to make layers with different number of keys.
```

Comment:
You mentionned in a comment you have removed it, from where it was used I assume. Might as well remove it from here then ?
Unless you think it can be on some use on keyboard that extends the en_keybord one and use the en_popup.

Reply:
Forgot about this one! Thanks!
No, I don't think it would be useful. It is easier to just enter `""`, I must have over thought this xD